.github/workflows/build-binaries.yaml

```
@@ -0,0 +1,146 @@
+        type: string
+      architecture:
+        required: true
+        type: string
+      cachix_cache_name:
+        required: true
+        type: string
+        description: "Cachix cache name to use"
+      build_file:
+        required: true
+        type: string
+        default: "Cargo.toml"
```

actionlint (1.7.9)

input "build_file" of workflow_call event has the default value "Cargo.toml", but it is also required. if an input is marked as required, its default value will never be used

(events)